waiter/bin/run-using-shell-scheduler.sh

```diff
@@ -21,7 +21,7 @@ if [ "${RECOMPILE}" != "0" ]; then
   lein do clean, compile
 fi
 
-WAITER_AUTH_RUN_AS_USER=$(id -un) lein run config-shell.edn
+WAITER_AUTH_RUN_AS_USER=$(id -un) lein run config-composite.edn
```

Comment:
I have the shell scheduler tests use the composite scheduler as it is effectively running the shell scheduler, but also enables a single specific test (`waiter.basic-test/test-composite-scheduler-services`) for the composite scheduler.

Reply:
We should still have a vanilla shell scheduler config.

Reply:
`config-shell.edn` continues to exist and undergoes basic validation in a unit test. Using `config-composite.edn` allows us to test the composite scheduler with one component shell scheduler.